src/seidr.py

```diff
@@ -47,20 +51,23 @@ def __init__(self,
         self.epsilon = epsilon
         # fatality rate
         self.alpha = alpha
+        # possibility of losing immunity
```

Comment:
Technically, I think "rate" is a better word then "possibility", since this model is deterministic.